pkg/skaffold/build/jib/sync.go

```diff
@@ -89,3 +221,14 @@ func (sm SyncMap) addEntries(entries []JSONSyncEntry, direct bool) error {
 	}
 	return nil
 }
+
+func toAbs(f string) (string, error) {
+	if !filepath.IsAbs(f) {
```

Comment:
Can we reuse this util function ?https://github.com/GoogleContainerTools/skaffold/blob/2ba2af8060e4f32684f9f53989039b7f7588aeca/pkg/skaffold/util/util.go#L238

Reply:
In jib, files can be outside of the workspace potentially.